bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,28 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
```

Comment:
-1 - i just fixed the whole thing, but we need this. the buildout cache is not the pip cache.